src/pages/patch/patchRestartModal/DownstreamProjectAccordian.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
```

Comment:
Is this component meant to be utilized in the patchRestartModal? If not, I think we should find another pathname for it

Reply:
moved